client/src/main/java/org/jboss/fuse/mvnd/client/DaemonConnector.java

```diff
@@ -232,7 +232,7 @@ public DaemonClientConnection startDaemon() {
                 return daemonConnection;
             }
             try {
-                sleep(200L);
+                sleep(10L);
```

Comment:
While the daemon is created, the registry does not contain any info, so the client would loop and read the registry continuously. But reading the registry involves complex file operations locking, so we need to let the daemon some space to actually write its own state to the registry.
It would be interesting to know how many iterations it takes on fast laptops.

Reply:
@ppalaga it's usually around 900ms / 1000ms to connect to a new daemon (the time between the `start` in this loop and the return statement), so it may be safer to revert to 200L to not consume too much cpu on not too fast computers.